steps/post-jira-comment-with-build-details-step/0.1.0/step.yml

```diff
@@ -0,0 +1,72 @@
+title: Create JIRA comment with build
+summary: Will post a comment on the issue page with build details (download link,
+  build number, etc)
+website: https://github.com/dag-io/post-jira-comment-with-build-details-step
+source_code_url: https://github.com/dag-io/post-jira-comment-with-build-details-step.git
+support_url: https://github.com/dag-io/post-jira-comment-with-build-details-step/issues
+published_at: 2016-03-29T12:30:43.404556034+02:00
+source:
+  git: https://github.com/dag-io/post-jira-comment-with-build-details-step.git
+  commit: 8d108e3d9117abaa1910110f15276d50679d3ad3
+type_tags:
+- script
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- git_branch: null
+  opts:
+    is_dont_change_value: true
+    is_expand: false
+    is_required: true
+    summary: ""
+    title: Git branch name
+    value_options: []
+- jira_user: null
+  opts:
+    is_dont_change_value: true
+    is_expand: false
+    is_required: true
+    summary: JIRA account username that will be the author of the comment
```

Comment:
Input's `description` will be visible on bitrise.io, but summary will not. If you want to describe the input please provide a description as well, but it is not required.